PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install CrowdStrike Falcon Agent.md

```diff
@@ -10,23 +10,71 @@ mac
 
 ```bash
 #!/bin/bash
-DownloadUrl="https://path/to/url.pkg"
+CSBaseAddress=""
```

Comment:
Minor thing. Maybe we can put a comment above the variables to be edited that says, `#### Edit these variables`